hypertools/tools/cluster.py

```diff
@@ -1,11 +1,15 @@
 #!/usr/bin/env python
 
 from sklearn.cluster import KMeans
-from sklearn.decomposition import PCA
+import warnings
 import numpy as np
 from .._shared.helpers import *
+from .reduce import reduce as reducer
+from .align import align as aligner
+from .normalize import normalize as normalizer
 
-def cluster(x, n_clusters=8, ndims=None):
+def cluster(x, n_clusters=8, ndims=None, model='IncrementalPCA',
```

Comment:
good point.  how about renaming those flags after their respective functions?  so:
- rename `model` to `reduce`
- `align` stays the same, but supports `True`, `False` (which defaults to aligning with hyperalignment or doing nothing, respectively), or `SRM` (align using SRM) or `hyperalign` (align using hyperalignment)-- default: `False`
- `cluster` stays the same, but supports `True` (align using k-means), `False` (do nothing; default), and `'kmeans'` (align using k-means) -- and possibly other models in the future

example syntax:
`y = hyp.plot(x, reduce='TSNE', cluster='kmeans', align='SRM', ndims=5)`

now `y` is an SRM-aligned TSNE-reduced 5-dimensional version of `x`, and a 3D plot (of `y`) is displayed.